diff --git a/apps/web/src/app/admin-console/organizations/organization-routing.module.ts b/apps/web/src/app/admin-console/organizations/organization-routing.module.ts
--- a/apps/web/src/app/admin-console/organizations/organization-routing.module.ts
+++ b/apps/web/src/app/admin-console/organizations/organization-routing.module.ts
@@ -17,6 +17,7 @@ import { deepLinkGuard } from "../../auth/guards/deep-link/deep-link.guard";
 
 import { VaultModule } from "./collections/vault.module";
 import { organizationPermissionsGuard } from "./guards/org-permissions.guard";
+import { mandatoryAuthenticatorGuard } from "../../vault/guards/mandatory-authenticator.guard";
 import { organizationRedirectGuard } from "./guards/org-redirect.guard";
 import { OrganizationLayoutComponent } from "./layouts/organization-layout.component";
 import { GroupsComponent } from "./manage/groups.component";
@@ -27,6 +28,7 @@ const routes: Routes = [
     path: ":organizationId",
     component: OrganizationLayoutComponent,
     canActivate: [deepLinkGuard(), authGuard, organizationPermissionsGuard(canAccessOrgAdmin)],
+    canActivateChild: [mandatoryAuthenticatorGuard],
     children: [
       {
         path: "",
